presto-spi/src/main/java/com/facebook/presto/spi/block/BlockUtil.java

```diff
@@ -34,11 +34,11 @@ private BlockUtil()
     {
     }
 
-    static void checkValidPositionsArray(int[] positions, int offset, int length)
+    static void checkValidArrayOffsets(int[] array, int offset, int length)
```

Comment:
`checkValidArrayOffsets` is not very meaningful too. Maybe `checkValidArrayRange` or `checkValidSubarray`?

Reply:
`checkValidArrayRange` sounds good